web-src/src/components/PromptSessionSideView.js

```diff
@@ -88,7 +90,14 @@ export function PromptSessionSideView({
         ? <Flex UNSAFE_className={styles.promptFlexItems} UNSAFE_style={{ borderBottom: '1px solid rgb(224, 224, 224)' }} direction={'column'} justifyContent={'stretch'} alignItems={'stretch'} gridArea={'info'}>
           <Flex UNSAFE_style={{ borderRadius: '8px', background: '#E0F2FF', padding: '10px' }} gap={'size-100'} alignItems={'center'}>
             <GenAIIcon />
-            <Text UNSAFE_className={styles.promptName}>{currentSession.name ?? 'Empty'}</Text>
+            <Text UNSAFE_className={styles.promptName}>{`${currentSession.name.split(' ')[0]} ${new Date(currentSession.name).toLocaleString(user.locale, {
```

Comment:
I recommend moving this to a constant.
Why is this needed `${currentSession.name.split(' ')[0]`?